internal/cmd/plugin/status/status.go

```diff
@@ -280,7 +280,7 @@ func (fullStatus *PostgresqlStatus) printBasicInfo(ctx context.Context, k8sClien
 		// Avoid printing the start time when hibernated or fenced
 		primaryStartTime := getPrimaryStartTime(cluster)
 		if len(primaryStartTime) > 0 {
-			summary.AddLine("Primary start time:", primaryStartTime)
+			summary.AddLine("Primary promotion time:", primaryStartTime)
```

Comment:
Why not add start/promotion time string? because it's both right? I haven't check yet the full code

Reply:
The primary could technically restart and the time wouldn't change AFAIK (that's why it isn't reliable as start time)